### PR TITLE
split NO

### PR DESCRIPTION
**Pull request by maxbellec**

### CODE REVIEW

**Comment by maxbellec** [comment]
screenshot at 2018-05-09 16-08-30

**Comment by corradio** [comment]
Test failing:
```
======================================================================
FAIL: test_bounding_boxes (test_zones_json.ZonesJsonTestcase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/circleci/electricitymap/parsers/test/test_zones_json.py", line 14, in test_bounding_boxes
    self.assertLess(bbox[0][0], bbox[1][0])
AssertionError: 199.5 not less than -199.5
```

File: config/exchanges.json [hunk 1]
```diff
@@ -1415,32 +1396,12 @@
...
   "NL->NO-NO2": {
     "parsers": {
       "exchangeForecast": "ENTSOE.fetch_exchange_forecast",
       "exchange": "statnett.fetch_exchange"
     }
   },
-  "NO->SE": {
-    "parsers": {
-      "exchange": "statnett.fetch_exchange",
-      "exchangeForecast": "ENTSOE.fetch_exchange_forecast"
```
Comment (corradio) [inline config/exchanges.json]: @maxbellec we still want to keep the exchange forecast here. Maybe ENTSOE has forecasts on BZ level?

**Comment by jarek** [comment]
I'll mention it here so that it doesn't get lost: the split ended up resetting Norway's hydro carbon intensity to the default 24 gCO2eq/kWh (because the production is now in e.g. NO-NO2 rather than NO)

**Pull request merged**